src/App.tsx

```diff
@@ -17,18 +53,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                filterBy={filterBy}
+                setFilterBy={setFilterBy}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {loading && (
+                <Loader />
+              )}
+
+              {!loading && todos.length > 0 && (
```

Comment:
```suggestion
              {!loading && !!todos.length && (
```

Reply:
Done. But why this `!!todos.length` better then this `todos.length > 0`?